docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='using-standalone-perspectives-standalone-custom-pages-proc']
+= Using standalone custom pages(dashboards)
+
+Apart from standalone perspectives, you can also embed custom pages (also known as dashboards) in your application. For accessing the custom pages from your application, provide the name of the custom page as the value of the `perspective` parameter. Note that the `perspective` parameter is case-sensitive.
+
+.Procedure
+. Login to {CENTRAL}.
+. In a web browser, enter the link of the custom page in the address bar, for example,
```

Comment:
instead of link, consider web address. Also, I looked up "URL" in the style guide and it says to avoid it.

Basically, it says "either use "web address" or do not use a term at all. For example, write "go to".. `http://localhost:8080/business-central/kie-wb.jsp?standalone=true&perspective=LibraryPerspective&header=UberfireBreadcrumbsContainer

Reply:
Changed partially.
Michele, I have changed "web link" to "web address" but have not rephrased the sentences using "go to ..." for readability consistency.